CustomSkillForDataIngestion/QnAIntegrationCustomSkill/QnAIntegrationCustomSkill.cs

```diff
@@ -125,7 +128,7 @@ public static class UploadToQnAMaker
             var stopwatch = new Stopwatch();
```

Comment:
unused ?

Reply:
The time taken by update operation is logged